AsyncDisplayKit/Private/_ASPendingState.mm

```diff
@@ -106,6 +110,30 @@ @implementation _ASPendingState
   ASPendingStateFlags _flags;
 }
 
+/**
+ * Apply the state's frame, bounds, and position to layer. This will not
+ * be called on synchronous view-backed nodes which require we directly
+ * call [view setFrame:].
+ *
+ * FIXME: How should we reconcile order-of-operations between setting frame, bounds, position?
+ * Note we can't read bounds and position in the background, so we have to keep the frame
+ * value intact until application time (now).
+ */
+ASDISPLAYNODE_INLINE void ASPendingStateApplyMetricsToLayer(_ASPendingState *state, CALayer *layer) {
```

Comment:
This name seems a bit too general, almost like it would trigger applying the entire pending state.  It is awesome to have this code separated out and shared, however.  Maybe ApplyGeometry ?